### DISCUSSION TITLE
확인이 필요한 이슈 목록 정리

### DISCUSSION THREAD

**Discussion by JeongJaeSoon** [2P3S/scrum-dice#97] [Ideas]
- [ ] https://github.com/2P3S/2p3s-backend-system/issues/50
- [ ] https://github.com/2P3S/2p3s-backend-system/issues/52

**Comment by Samsam-lee** [comment]
- [ ] Navigation 에서 언어 선택 시 특정 페이지로 이동하는데 언어 선택 버튼이 모든 페이지에서 보이는게 맞는지 검토
- [ ] 페이지 순서 부분 한 번 더 얘기를 나눠보기
